lib/Image/Chromium.php

```diff
@@ -48,18 +62,20 @@ public static function getChromiumBinary(): ?string
      */
     public static function convert(string $url, string $outputFile, ?string $sessionName = null, ?string $sessionId = null, string $windowSize = '1280,1024'): bool
     {
-        $binary = self::getChromiumBinary();
-        if (!$binary) {
-            return false;
+        try {
+            $browser = BrowserFactory::connectToBrowser(\Pimcore\Config::getSystemConfiguration('chromium')['base_url']);
```

Comment:
i have some perfomance doubts, could it be slow to check if the websocket exists? is there any more native or just to sort of "ping"? or should it be that whenever the binary chromium is not found, we expect and check that the docker container eg https://github.com/pimcore/pimcore/pull/14457/files?w=1#diff-48c7411580a8efd2225afdf0851c253e959b509e1437cb27611c1418f49c8828R34-R45?